practice.md

```diff
@@ -21,15 +21,16 @@ To the extent 18F has the rights to do so, 18F will normalize the copyright stat
 
 ### Contributing back to outside projects
 
-18F staff are encouraged to contribute back any modifications or improvements they make to open source software projects outside 18F -- whether government or non-government -- in the course of their work. When 18F staff begin modifications to outside work, they should plan with eventual upstream contribution in mind.
+18F staff are encouraged to seek existing, open source solutions -- whether government or non-government -- before writing custom tools. When existing libraries need to be modified or improved, 18F staff should make the modifications with eventual upstream contribution in mind.
+
+In practice, this generally involves forking the relevant repository to the 18F organization within GitHub, creating a new branch with the modifications, and sending a pull request to upstream from the 18F fork. Unlike our own projects, there is no need for internal code review in this scenario (though it doesn't hurt).
```

Comment:
FWIW, I have done it both ways (forking to 18F org, or forking to my individual account) depending upon whether the project expects the upstream PR to be merged and how quickly. Forking to the 18F org creates an ongoing dependency for 18F to maintain, so I have preferred *not* to do that, unless the repo represents an important dependency for a 18F project (e.g. SAML libs for login.gov).

So, I don't know that I would always recommend forking to 18F org. I think it depends on whether 18F intends to "own" the fork in perpetuity, unless we have different rules around deleting forks of repos vs original repos?